java-11-sample/pom.xml

```diff
@@ -27,23 +27,22 @@
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-starter</artifactId>
         </dependency>
+        <dependency>
+			<groupId>org.springframework.cloud</groupId>
```

Comment:
nit: indentation is not aligned with rest of the file?

please also check indentations in other files.

Reply:
Fixed